tox.ini

```diff
@@ -250,3 +250,16 @@ commands =
 whitelist_externals =
     docker-compose
 passenv = DOCKER_*
+
+[testenv:integration]
+commands =
+    {[base]pip_install} acme . certbot-nginx certbot-ci
+    pytest {toxinidir}/certbot-ci/certbot_integration_tests \
+        --acme-server={env:ACME_SERVER:pebble-nonstrict} \
+        --cov=acme --cov=certbot --cov=certbot_nginx --cov-report= \
+        --cov-config={toxinidir}/certbot-ci/certbot_integration_tests/.coveragerc \
+        -W 'ignore:Unverified HTTPS request'
+    coverage report --fail-under=65 --show-missing
+passenv =
+    TRAVIS
```

Comment:
where is this being used in the code?

Reply:
Old fix because because pytest-xdist needs to know when it is on Travis, because Travis CI is declaring an insane number of VCPUs (32) instead of the real number (2). Yes, it is hardcoded in pytest-xdist.

But this is not needed anymore, since another general fix I did, to set `TRAVIS` and `APPVEYOR` as passing by default to any tox env when executed from theses CI.

I remove